<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentationWidget</class>
 <widget class="QDockWidget" name="SegmentationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>218</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>543</width>
    <height>218</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>543</width>
    <height>524287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Segmentation</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBoxChannels">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Channel selector</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioGrey">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Grey</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioRed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioGreen">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioBlue">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
      <property name="spacing">
       <number>-1</number>
      </property>
      <item>
       <widget class="QLabel" name="labelNotCreated">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>119</red>
              <green>119</green>
              <blue>119</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Press Create New to create an object!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBase">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Base object:</string>
        </property>
        <property name="margin">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboSourceObject">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButtonBaseGo">
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelLowThreshold">
        <property name="text">
         <string>Low threshold:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalLowSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxLowThreshold">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="labelHighThreshold">
        <property name="text">
         <string>High threshold:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalHighSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxHighThreshold">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButtonUpdate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxAutoUpdate">
        <property name="text">
         <string>Auto update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNew">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Create New</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalLowSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxLowThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxLowThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalLowSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalHighSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxHighThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxHighThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalHighSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
